pkgs/development/python-modules/pytest-sanic/default.nix

```diff
@@ -1,28 +1,33 @@
 { lib
+, aiohttp
+, async_generator
 , buildPythonPackage
 , fetchPypi
+, httpx
 , pytest
-, aiohttp
-, async_generator
+, websockets
 }:
 
 buildPythonPackage rec {
   pname = "pytest-sanic";
-  version = "1.6.2";
+  version = "1.7.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "6428ed8cc2e6cfa05b92689a8589149aacdc1f0640fcf9673211aa733e6a5209";
+    sha256 = "sha256-/eLC/ZbZzvXq0EJd/gl1EaJXI3RrYTImmg6/dk6Np0I=";
   };
 
   propagatedBuildInputs = [
-    pytest
     aiohttp
     async_generator
+    httpx
+    pytest
+    websockets
   ];
 
   # circular dependency on sanic
```

Comment:
Maybe we can resolve this circular dependency by using a sanic without tests?

Reply:
Is worth a try.

Reply:
I just catched a strange bug with a similar case. This only works if sanic is a test dependency in both packages and not a hard dependency. If it is not the package is not correctly installed and we end up with a mostly empty package in nix.

Reply:
At first glance it looks like it works but I didn't check the content of the packages.